configure.ac

```diff
@@ -640,6 +640,12 @@ AC_CHECK_FUNCS(explicit_bzero memset_explicit)
 AC_CHECK_FUNCS([ruserok_af ruserok], [break])
 AC_CHECK_FUNCS(close_range)
 
+dnl For module/pam_timestamp
+AC_CHECK_HEADERS([sys/random.h unistd.h])
```

Comment:
unistd.h is already listed earlier, and it's indeed long assumed to be universally available.

Reply:
Removed it. Had it there for completeness. That check was cached anyway though.